homeassistant/components/shelly/strings.json

```diff
@@ -3,7 +3,7 @@
     "flow_title": "{name}",
     "step": {
       "user": {
-        "description": "Before set up, battery-powered devices must be woken up, you can now wake the device up using a button on it.",
+        "description": "Before setup, battery-powered devices must be woken up, you can now wake the device up using a button on it.",
```

Comment:
I think "setup" would fit here better, we do refer here to the setup process not the act itself of setting up a device.

Reply:
Isn't that what I changed it to?

Reply:
Sorry, looked at wrong line since most of them was change from `setup` to `set up`.